DeleteMiddleNode.py

```diff
@@ -0,0 +1,36 @@
+class ListNode:
+    def __init__(self, value=0, next=None):
+        self.value = value
+        self.next = next
+
+def deleteNode(node):
+    if node is None or node.next is None:
+        return
+    nextNode = node.next
```

Comment:
Although Python will let you name variables pretty much whatever you want, the naming convention for this language is to use snake case instead of camel case (e.g. `next_node` instead of `nextNode`).